.github/workflows/code.yml

```diff
@@ -9,6 +9,7 @@ on:
     branches:
     - main
     paths:
+    - "**"
```

Comment:
Markdown and yaml files are excluded in the following lines. It is basically the inverse of the [previously used paths-ignore](https://github.com/anexia-it/go-anxcloud/pull/97/files#diff-2d2301c62da7f04adcadfbd43fa4063e9ad651296b6f190974b99e72a00017a2L11-L16).

We inverted it because paths-ignore does not support negative matches which are required to allow changes to the CHANGELOG.md & workflow files.